command/ca/provisionerbeta/update.go

```diff
@@ -794,3 +815,32 @@ func updateGCPDetails(ctx *cli.Context, p *linkedca.Provisioner) error {
 	}
 	return nil
 }
+
+func updateSCEPDetails(ctx *cli.Context, p *linkedca.Provisioner) error {
+	data, ok := p.Details.GetData().(*linkedca.ProvisionerDetails_SCEP)
+	if !ok {
+		return errors.New("error casting details to SCEP type")
+	}
+	details := data.SCEP
+
+	if ctx.IsSet("force-cn") {
+		details.ForceCn = ctx.Bool("force-cn")
+	}
+	if ctx.IsSet("challenge") {
+		details.Challenge = ctx.String("challenge")
+	}
+	if ctx.IsSet("capabilities") {
+		details.Capabilities = ctx.StringSlice("capabilities")
+	}
+	if ctx.IsSet("minimum-public-key-length") {
```

Comment:
don't forget to change this to `min` if you change the flag name

Reply:
Fixed in 8606fdcf0560e56688309948f5c81d23f8588022.